third_party/2and3/pymysql/cursors.pyi

```diff
@@ -26,6 +28,8 @@ class Cursor:
     def fetchall(self) -> Optional[Tuple[Gen, ...]]: ...
     def scroll(self, value: int, mode: str = ...): ...
     def __iter__(self): ...
+    def __enter__(self: _SelfT) -> _SelfT: ...
+    def __exit__(self, *exc_info: Tuple[Any]): ...
```

Comment:
`Tuple` is already implied by the star. Therefore this would expect arguments to be tuples.

Also, this should return `None`.